torch/_decomp/decompositions.py

```diff
@@ -1189,6 +1189,19 @@ def addmm(self: Tensor, mat1: Tensor, mat2: Tensor, beta: int = 1, alpha: int =
     return out + beta * self
 
 
+@register_decomposition(aten._addmm_activation)
```

Comment:
Wonder why do we need this decomposition pass? It seems to me that when max_auto_tune is false, we do fusion via the pattern-matching fx pass. When max_auto_tune is True, fusion is not done and we also don't need this decomposition pass, correct?

Reply:
This is registering the decomposition so that we get meta support with symbolic sizes, but it's not enabling it for torch/_inductor. To be enabled it would need to be in the list [here](https://github.com/pytorch/pytorch/blob/main/torch/_inductor/decomposition.py#L40) or in core_aten_decomps.

We reuse the decompositions for fake tensor meta kernel propagation. Since there's already an addmm aten decomp, adding addmm_activation decomp is easiest way to get meta kernel support.

Reply:
I see, thanks @eellison !